src/components/Form/Checkbox.tsx

```diff
@@ -136,6 +142,8 @@ export class Checkbox extends Component<CheckboxProps> {
       checked: !!checkedState,
       className: checkboxBaseClassNames.input,
       onChange: this.handleClick,
+      name,
```

Comment:
Since name is optional, this could results into name="" in DOM. Probably no side effects though.

Reply:
According to the snapshot test without the `name` prop, it does not end as `name=""` in the DOM.